tests/test_mutations.py

```diff
@@ -1007,22 +1085,36 @@ def sign_tst(self, diffs):
 
 class TestMutationStatistics(StatisticalTestMixin):
     def verify_model(self, model, verify_roots=True, state_independent=False):
-        ots = msprime.simulate(10, random_seed=5, recombination_rate=0.05, length=20)
+        # NOTE: this test is pretty brittle and some will fail if
```

Comment:
I was thinking about this in my previous comment. Why is this brittle?

Reply:
It's very sensitive to the random seed - sometimes it works, but more often than not it doesn't. So, even if it works on the current random seed, it will fail mysteriously at some point down the line when GSL changes the RNG a bit.

TBH, I wonder if these tests should be unit tests at all, and they would be more appropriate as part of the verification suite.

Reply:
Hm. Moving them to verification makes sense - as long as we make sure to run verification if we change the mutation code in the future.

But, we should be able to make these not brittle... oh! by switching from a p-value condition to a confidence interval condition!

Reply:
https://github.com/tskit-dev/msprime/issues/1334